### PR TITLE
add 2 example usage

### PR DESCRIPTION
**Pull request by Ilite-Kiriya**

### CODE REVIEW

File: docs/examples/calc_jos3.html [hunk 1]
```diff
@@ -0,0 +1,205 @@
...
+      ></h>
+    </div>
+
+    <div>
+      <h2>EXAMPLE 2 (detail simulation)</h2>
+      <div id="graph2" style="width: 600px; height: 600px; margin: auto"></div>
+    </div>
+  </body>
+  <script type="module">
+    import { JOS3 } from "../../src/models/JOS3.js";
```
Comment (szaldivar) [inline docs/examples/calc_jos3.html:37]: import from CDN

File: docs/examples/calc_jos3.html [hunk 2]
```diff
@@ -0,0 +1,205 @@
...
+      "graph2",
+      [
+        t_skin_mean_trace,
+        t_skin_head_trace,
+        t_skin_chest_trace,
+        t_skin_left_hand_trace,
+      ],
+      layout,
+    );
+    model.to_csv("jos3_output_example/jos3_example2 (all output).csv");
```
Comment (szaldivar) [inline docs/examples/calc_jos3.html:203]: I think we can omit the csv part

File: docs/examples/calc_jos3.html [hunk 3]
```diff
@@ -0,0 +1,205 @@
...
+      xaxis: {
+        title: "Time [min]",
+      },
+      yaxis: {
+        title: "Mean skin temperature [°C]",
+      },
+    };
+    Plotly.newPlot("graph1", [t_skin_mean_trace], layout);
+
+    model.to_csv("jos3_output_example/jos3_example1 (default output).csv");
```
Comment (szaldivar) [inline docs/examples/calc_jos3.html:68]: we can omit this

File: docs/examples/jos3_output_example/human_subject_experiment_dataset.xlsx
Comment (szaldivar) [inline docs/examples/jos3_output_example/human_subject_experiment_dataset.xlsx:1]: we probably want to remove this

**Review by szaldivar — CHANGES_REQUESTED** [review]

File: docs/examples/calc_jos3.html [hunk 4]
```diff
@@ -0,0 +1,203 @@
...
+      yaxis: {
+        title: "Mean skin temperature [°C]",
+      },
+    };
+    Plotly.newPlot("graph1", [t_skin_mean_trace], layout);
+
+    // Print the BMR value using the getter
+    document.getElementById("example1 bmr").innerHTML = model.bmr;
+    document.getElementById("example1 Body name list").innerHTML =
+      model.body_names;
```
Comment (szaldivar) [inline docs/examples/calc_jos3.html:71]:
```suggestion
    model.body_names;
```
I want to test something

**Comment by FedericoTartarini** [comment]
@szaldivar could you please, if you have time, review this pull request and merge it if you are happy with the code?

**Comment by szaldivar** [comment]
> @szaldivar could you please, if you have time, review this pull request and merge it if you are happy with the code?

Hi @FedericoTartarini, sorry I have been super busy. Yes I will take a look, I will schedule some of my time next week to get this resolved

**Comment by szaldivar** [comment]
> @szaldivar could you please, if you have time, review this pull request and merge it if you are happy with the code?

Actually this looks good, ticked

**Comment by FedericoTartarini** [comment]
Thank you very much for working on this.

**Pull request merged**